ntoskrnl/config/cmsysini.c

```diff
@@ -647,13 +654,12 @@ CmpCreateControlSet(IN PLOADER_PARAMETER_BLOCK LoaderBlock)
                                KeyHandle,
                                NULL);
     Status = NtOpenKey(&ConfigHandle, KEY_READ, &ObjectAttributes);
-    NtClose(KeyHandle);
 
     /* Check if we don't have one */
     if (!NT_SUCCESS(Status))
     {
         /* Cleanup and exit */
-        ConfigHandle = NULL;
+        Status = STATUS_SUCCESS;
```

Comment:
I understand that this replicates the previous behavior of unconditionally returning STATUS_SUCCESS at the end, but I'm not convinced that this is the appropriate thing to do in all cases.

Reply:
I'm also not sure about that. The question is: How to reliably verify a change here? Just leaving out all STATUS_SUCCESS lines in failure cases, installing ReactOS, and if that works all is well?